lib/contact_fixer.rb

```diff
@@ -4,6 +4,18 @@ def initialize(contacts_api, output)
     @output = output
   end
 
+  def is_non_roman(str)
+    str =~ /[^\w\s!..]/
```

Comment:
Maybe make it static?

Reply:
Also, add a comment from where you pulled this magic / how does it work

Reply:
Done